### PR TITLE
Porting Oculus OpenVR fixes from XRTK to MRTK

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
## Overview
Porting over OpenVR fixes for the OculusTouch controllers, resolving controller mapping issues

## Changes
- Fixes: #5263

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset [hunk 1]
```diff
@@ -1144,9 +1144,9 @@ MonoBehaviour:
       axisType: 7
       inputType: 3
       inputAction:
-        id: 4
-        description: Pointer Pose
-        axisConstraint: 7
+        id: 0
+        description: None
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset]: Looks like the action mappings were lost when the keycodes / axes were updated.

File: Assets/MixedRealityToolkit.Providers/OpenVR/OculusTouchController.cs [hunk 2]
```diff
@@ -33,12 +33,12 @@ public class OculusTouchController : GenericOpenVRController
...
+            new MixedRealityInteractionMapping(4, "Axis1D.PrimaryIndexTrigger Press", AxisType.Digital, DeviceInputType.TriggerPress, KeyCode.JoystickButton14),
             new MixedRealityInteractionMapping(5, "Axis1D.PrimaryHandTrigger Press", AxisType.SingleAxis, DeviceInputType.Trigger, ControllerMappingLibrary.AXIS_11),
             new MixedRealityInteractionMapping(6, "Axis2D.PrimaryThumbstick", AxisType.DualAxis, DeviceInputType.ThumbStick, ControllerMappingLibrary.AXIS_1, ControllerMappingLibrary.AXIS_2),
             new MixedRealityInteractionMapping(7, "Button.PrimaryThumbstick Touch", AxisType.Digital, DeviceInputType.ThumbStickTouch, KeyCode.JoystickButton16),
             new MixedRealityInteractionMapping(8, "Button.PrimaryThumbstick Near Touch", AxisType.Digital, DeviceInputType.ThumbNearTouch, ControllerMappingLibrary.AXIS_15),
-            new MixedRealityInteractionMapping(9, "Button.PrimaryThumbstick Press", AxisType.Digital, DeviceInputType.ThumbStickPress, KeyCode.JoystickButton8),
+            new MixedRealityInteractionMapping(9, "Button.PrimaryThumbstick Press", AxisType.Digital, DeviceInputType.ThumbStickPress, KeyCode.JoystickButton16),
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Providers/OpenVR/OculusTouchController.cs]: Do these changes mean the mappings on Unity's docs are wrong? https://docs.unity3d.com/Manual/OpenVRControllers.html

File: Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset (hunk 1 above)
Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset]: Odd, Unity mustn't have saved the asset properly, GAH. Fixing

File: Assets/MixedRealityToolkit.Providers/OpenVR/OculusTouchController.cs (hunk 2 above)
Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit.Providers/OpenVR/OculusTouchController.cs]: Good catch, not sure why that updated. Will double check and update in XRTK too

**Comment by SimonDarksideJ** [comment]
Over to you @keveleigh

**Pull request merged**